NOM_visualizations/R/NOM_R_notebook.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>Using read_csv rather than read.csv will likely be faster!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Switched to read_csv, it still takes 40 minutes alas</p>